protected/modules/OphCiExamination/migrations/m170921_013223_patient_diagnosis_assignment_view.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+class m170921_013223_patient_diagnosis_assignment_view extends CDbMigration
+{
+    public function createView($view_name, $select)
+    {
+        $this->dbConnection->createCommand('create view ' . $view_name . ' as ' . $select)->execute();
+    }
+
+    public function dropView($view_name)
+    {
+        $this->dbConnection->createCommand('drop view ' . $view_name)->execute();
+    }
+
+    public function up()
+    {
+        $this->createView('diagnosis_examination_events', <<<EOSQL
+select event_date, event.created_date, event_id, patient_id from et_ophciexamination_diagnoses et
+join event on et.event_id = event.id
+join episode on event.episode_id = episode.id
+EOSQL
+        );
+        $this->createView('latest_diagnosis_examination_events', <<<EOSQL
+select t1.event_id, t1.patient_id from diagnosis_examination_events t1
```

Comment:
What happened to our FROM clauses being on separate lines? Is this the acrosshealth standard?